libsolidity/lsp/LanguageServer.h

```diff
@@ -107,4 +109,22 @@ class LanguageServer
 	Json::Value m_settingsObject;
 };
 
+// Helper base class for implementing handlers.
+class Handler
+{
+public:
+	explicit Handler(LanguageServer& _server):
+		m_server{_server},
+		m_charStreamProvider{_server.charStreamProvider()},
```

Comment:
If w estore the server, why do we need all these const refs?

Reply:
For quicker access. I can drop them.